src/mon/mon_types.h

```diff
@@ -473,6 +473,20 @@ namespace ceph {
 	  );
       }
 
+      /**
+       * All the features that require a full quorum to be enabled.
+       *
+       * Features should only be added to this list if you want to
+       * make sure they are only enabled automatically when the quorum
+       * is composed by *all* the monitors in the monmap.
+       */
+      constexpr mon_feature_t get_requires_full_quorum() {
+        return (
+          FEATURE_LUMINOUS |
+          FEATURE_NONE
+          );
+      }
+
```

Comment:
No. We can simply rely on the persistent features. Just thought that there could be some features we would want to set on a full quorum, others maybe not so, but yeah... I suppose all the persistent would be nice to set only when all the monitors belong to the quorum.